src/main/java/seedu/address/logic/parser/findcommandparser/FindMentorCommandParser.java

```diff
@@ -23,16 +27,23 @@
      */
     public FindMentorCommand parse(String args) throws ParseException {
         ArgumentMultimap argumentMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME);
+                ArgumentTokenizer.tokenize(
+                        args, PREFIX_NAME, PREFIX_EMAIL, PREFIX_PHONE, PREFIX_ORGANISATION);
+
+        Optional<String> name = argumentMultimap.getValue(PREFIX_NAME);
+        Optional<String> email = argumentMultimap.getValue(PREFIX_EMAIL);
+        Optional<String> phone = argumentMultimap.getValue(PREFIX_PHONE);
+        Optional<String> organization = argumentMultimap.getValue(PREFIX_ORGANISATION);
+
+        boolean allPrefixesEmpty = name.isEmpty() && email.isEmpty()
```

Comment:
Can also use `AlfredParserUtil#arePrefixesPresent()`

Reply:
Hmm It wouldnt work for my case, because I need any of them to match not all. I could write another function which uses `anyMatch`, but I needed the individual values also.

Reply:
Ohh isee i see